torch/distributed/rpc/__init__.py

```diff
@@ -176,7 +193,7 @@ def _validate_rpc_args(backend, store, name, rank, world_size, rpc_backend_optio
 
 
     def _init_rpc_backend(
-        backend=backend_registry.BackendType.TENSORPIPE,
+        backend=backend_registry.BackendType.TENSORPIPE,  # type: ignore
```

Comment:
s/backend_registry.// 
```suggestion
        backend=BackendType.TENSORPIPE,  # type: ignore[attr-defined]
```